src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -77,6 +77,8 @@ private static VersionInfo GetVersionInfo()
 
         public override Schema Schema => _bindings.Schema;
 
+        public Schema OutputSchema => Schema;
```

Comment:
>Schema [](start = 38, length = 6)

_binding.Schema?
Also I think this is indication of what something went wrong.
It work before without defining two schemas, right?
You move schema around and now you need to have two of them.
Maybe that `public override Schema Schema => _bindings.Schema;` should be renamed to OutputSchema?

Reply:
This is something I'd expect because ML.NET mixes the concepts of variables and operators. Some objects such as RowToRowMapperTransform, it is an operator so that it has an output schema and it also contains a schema because it's a data (IDataView).

---
In reply to: [237264904](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237264904) [](ancestors = 237264904)

Reply:
In that case, I might prefer that, given that we reverse this, taht is, since `RowToRowTransformBase` will eventually be deprecated (as it is using the now disfavored `IDataTransform` interface) that we have `OutputSchema` reference `_bindings.Schema`, and `Schema` above point to `OutputSchema`. Not a big deal, certainly not a blocker, but just something to consider if you're going to be making other changes anyway.

---
In reply to: [237271984](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237271984) [](ancestors = 237271984,237264904)

Reply:
Discussed with Tom offline. It turns out that we can use `new Schema OutputSchema { get;  }` to merge two identical fields in a derived interface.

---
In reply to: [237582692](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237582692) [](ancestors = 237582692,237271984,237264904)

Reply:
There are some places where OutputSchema returns Schema. I will fixed all those cases in another iteration.


---
In reply to: [237686111](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237686111) [](ancestors = 237686111,237582692,237271984,237264904)